appveyor.yml

```diff
@@ -0,0 +1,47 @@
+image: Visual Studio 2017
+install:
+- ps: >-
+    Install-Product node LTS x64
+
+    choco install jq
+
+    bash ./get_repo.sh
+
+    $env:LATEST_MS_TAG="$(git -C vscode describe --tags)"
+
+    git -C vscode revert 90c13ff d1fed9e --no-commit
```

Comment:
Not just yet.  1.28.0 was branched a couple days before the fix went in.  The issue was tagged for being included in the October release, so I'm hoping 1.28.1?  It may be worthwhile just to wait a little while until it gets tagged.

Reply:
ahhhhh that's sad. yeah maybe we can wait to merge until they tag.